tests/e2e/matmul/BUILD.bazel

```diff
@@ -70,6 +70,10 @@ py_binary(
         "--lhs_rhs_type=%s" % lhs_rhs_type,
         "--shapes=large",
     ],
+    tags = [
+        # "--shapes=large" makes this test noriscv.
+        "noriscv",
```

Comment:
similar nit: I'd put the "can cause timeouts" context here

"tests using large shapes can timeout on riscv emulators"